### ISSUE TITLE
[Bug]: Click `Add a card` show long blank board

### ISSUE THREAD

**Issue by luke396** [mgmeyers/obsidian-kanban#973]
### Describe the bug

This is what appears when clicking ‘Add a card’.

It may not be a bug, but possibly due to some other configuration conflict. However, I’m unsure. Apologies for the inconvenience.

<img width="235" alt="image" src="https://github.com/mgmeyers/obsidian-kanban/assets/77875500/9df83ffe-ef7d-42f9-95b7-de225d328905">

### Steps to reproduce

 My machine is running Windows 11, and I am using the latest version of Obsidian, along with the Kanban plugin.

The issue with Obsidian’s theme appears to be irrelevant, as I have experimented with different themes.

Apologies for the confusion.

### Operating system

Windows

**Comment by mgmeyers** [comment]
@luke396 does this happen with the default theme? You might also try disabling other plugins and enabling them 1 by 1 to see if any of them conflict.

**Comment by luke396** [comment]
Hi, @mgmeyers. Thanks for your guidance. I disabled and re-enabled all plugins individually and identified the issue. It seems to be a conflict between Kanban and [cm-typewriter-scroll-obsidian](https://github.com/deathau/cm-typewriter-scroll-obsidian), possibly due to recent updates to Kanban since the two plugins have coexisted for quite some time.

In any case, I've uninstalled the other plugin. I really appreciate Kanban, and I'm grateful for your development and debugging tips!
